apps/sms/style/attachment.css

```diff
@@ -164,6 +185,15 @@
   background-position: left -30rem;
 }
 
+.vcard-placeholder {
+  background: url('images/attachment-vcard.png') no-repeat left top / 4rem;
+}
+
+/* Size is not shown in ref (vcard) attachments */
+div.vcard-placeholder ~ div.size-indicator {
```

Comment:
Maybe you can remove this and modify the original selector `.attachment-draft  .size-indicator` into `.attachment-draft > :not([data-attachment-type="vcard"]) .size-indicator`